syncd/sal.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
i posted PR to add common header https://github.com/Azure/sonic-swss-common/pull/320, when that will be merged, i will remove this file here, i even can wait with merging this until common pr will be merged

Reply:
That one is merged. You can go ahead remove this one.

---
In reply to: [353245701](https://github.com/Azure/sonic-sairedis/pull/541#discussion_r353245701) [](ancestors = 353245701)

Reply:
done